pandas-stubs/io/sql.pyi

```diff
@@ -143,7 +144,7 @@ def read_sql(
 ) -> DataFrame: ...
 
 class PandasSQL:
-    def read_sql(self, *args, **kwargs): ...
```

Comment:
This does not exist anymore and below we needed to add abstractmethod decorator.

Reply:
Okay will revert here for now, seems like `PandasObject` which is what it inherits from is not yet documented in the stubs so will hold.